src/common/routes.js

```diff
@@ -14,5 +15,6 @@ export default (
     <Route path="/:owner/:name/builds" component={Builds}/>
     <Route path="/:owner/:name/builds/:buildId" component={BuildDetails}/>
     <Route path="/login/failed" component={LoginFailed}/>
+    <Route path="/dashboard/select-repositories" component={SelectRepositories}/>
```

Comment:
@bartaz: We don't necessarily have to have separate routes for "logged in" and "logged out" versions of the dashboard. If we go with `dashboard/add`, `katiefenn/dashboard/add` or `katiefenn`, we could write an Express handler to redirect requests to `/` in the absence of a GitHub auth key.

This gives us the ability to railroad users into the landing page as well as giving us URL state.